dev/sql/test_data.sql

```diff
@@ -96,16 +96,25 @@ COMMIT;
 INSERT INTO poll_t (
     id,
     title,
-    method,
+    config_id,
     visibility,
     state,
     sequential_number,
     content_object_id,
     meeting_id
 )
-VALUES (1, 'Titel1', 'selection', 'open', 'created', 1, 'topic/1', 2);
+-- TODO: use the correct id from poll_config_approval.
+VALUES (1, 'Titel1', 'poll_config_approval/1', 'open', 'created', 1, 'topic/1', 2);
 SELECT nextval('poll_t_id_seq');
 
+INSERT INTO poll_config_approval_t (
+    poll_id,
+    allow_abstain
+)
+-- TODO: Use correct IDs. Can be removed after this is fixed: https://github.com/OpenSlides/openslides-meta/issues/339
+VALUES (1, True);
+
+
```

Comment:
@hjanott I as not able to fix this.

It creates the Error: " psql:../sql/test_data.sql:107: ERROR:  insert or update on table "poll_t" violates foreign key constraint "poll_t_config_id_poll_config_approval_id_fkey"
DETAIL:  Key (config_id_poll_config_approval_id)=(1) is not present in table "poll_config_approval_t"."

Could you either fix this or allow me to remove the poll from the test data?